gems/pending/util/miq_apache/miq_apache.rb

```diff
@@ -206,7 +206,8 @@ def self.create_balancer_config(opts = {})
     def self.create_redirects_config(opts = {})
       opts[:redirects].to_miq_a.each_with_object("") do |redirect, content|
         if redirect == "/"
-          content << "RewriteRule ^/self_service(?!/(assets|images|img|styles|js|fonts|bower_components|gettext)) /self_service/index.html [L]\n"
+          content << "RewriteRule ^/ui/service(?!/(assets|images|img|styles|js|fonts|vendor|gettext))
+                      /ui/service/index.html [L]\n"
```

Comment:
@AllenBW If the line splitting was just to appease rubocop, then let's put it back to single line just to be safe.